release.nix

```diff
@@ -225,10 +225,12 @@ let
       nix = build.x86_64-linux; system = "x86_64-linux";
     });
 
-    tests.setuid = pkgs.lib.genAttrs (pkgs.lib.filter (pkgs.lib.hasSuffix "-linux") systems) (system:
-      import ./tests/setuid.nix rec {
-        nix = build.${system}; inherit system;
-      });
+    tests.setuid = pkgs.lib.genAttrs
+      (pkgs.lib.filter (system: system == "x86_64-linux" || system == "i686-linux") systems)
```

Comment:
This can be written more concisely as `genAttrs ["i686-linux" "x86_64-linux"]`. :-)